README.md

```diff
@@ -10,7 +10,7 @@ Setup may vary by environment.
 
 ## React Apps
 
-See [packages/react-components](https://github.com/Agoric/ui-kit) for convenient hooks and utilities that make setting up
+See [packages/react-components](https://github.com/Agoric/ui-kit/tree/main/packages/react-components) for convenient hooks and utilities that make setting up
```

Comment:
Unfortunately, this means when you're reading the [generated docs](https://samsiegart-patch-1.ui-kit-dwm.pages.dev/), this link takes you out of them and back the github source view.

ideally, the link would go to https://samsiegart-patch-1.ui-kit-dwm.pages.dev/modules/_agoric_react_components when looking at the generated docs. But I don't see a straightforward way to do that.

One option would be

```
See the `@agoric/react-components` package for utilities such as {@link AgoricProvider} ...
```

but that link wouldn't work from within github.